custom/icds_reports/utils/data_pull/governance_dashboard.sql

```diff
@@ -0,0 +1,177 @@
+-- to pull the vhnd data submitted in November
+SELECT
+    state_name,
+    district_name,
+    block_name,
+    supervisor_name,
+    awc_name,
+    '2019-11-01' as month,
+    1 as vhsnd_conducted,
+    vhsnd_date_past_month,
+    child_immu,
+    anc_today,
+    vhnd_gmp
+FROM agg_awc_monthly full join "ucr_icds-cas_static-vhnd_form_28e7fd58" ucr
+    ON (ucr.awc_id = agg_awc_monthly.awc_id and ucr.vhsnd_date_past_month>='2019-11-01' and ucr.vhsnd_date_past_month<'2019-12-01')
+where agg_awc_monthly.num_launched_awcs=1 and agg_awc_monthly.month='2019-11-01' and agg_awc_monthly.aggregation_level=5;
+
+
+/*
+ Nested Loop  (cost=1000.85..203493.44 rows=6 width=128)
+   ->  Nested Loop Left Join  (cost=1000.85..203468.38 rows=1 width=96)
+         ->  Gather  (cost=1000.42..203467.54 rows=1 width=111)
+               Workers Planned: 4
+               ->  Nested Loop  (cost=0.42..202467.44 rows=1 width=111)
+                     ->  Parallel Append  (cost=0.00..113228.11 rows=141670 width=173)
+                           ->  Parallel Seq Scan on "agg_awc_2019-11-01_5" agg_awc_1  (cost=0.00..112519.76 rows=141670 width=173)
+                                 Filter: ((month = '2019-11-01'::date) AND (aggregation_level = 5) AND (num_launched_awcs = 1))
+                           ->  Parallel Seq Scan on agg_awc  (cost=0.00..0.00 rows=1 width=168)
+                                 Filter: ((month = '2019-11-01'::date) AND (aggregation_level = 5) AND (num_launched_awcs = 1))
+                     ->  Index Scan using awc_location_local_doc_id_idx on awc_location_local awc_location  (cost=0.42..0.62 rows=1 width=240)
+                           Index Cond: (doc_id = agg_awc_1.awc_id)
+                           Filter: ((aggregation_level = 5) AND (agg_awc_1.state_id = state_id) AND (agg_awc_1.district_id = district_id) AND (agg_awc_1.block_id = block_id) AND (agg_awc_1.supervisor_id = supervisor_id))
+         ->  Index Scan using "ix_ucr_icds-cas_static-vhnd_form_28e7fd58_awc_id" on "ucr_icds-cas_static-vhnd_form_28e7fd58" ucr  (cost=0.43..0.84 rows=1 width=49)
+               Index Cond: (awc_id = awc_location.doc_id)
+               Filter: ((vhsnd_date_past_month >= '2019-11-01'::date) AND (vhsnd_date_past_month < '2019-12-01'::date))
+   ->  Seq Scan on icds_months_local months  (cost=0.00..25.00 rows=6 width=4)
+         Filter: (start_date = '2019-11-01'::date)
+(18 rows)
+
+*/
+
+
+
+-- to get children open and valid in oct and Nov in single query. It is needed in single query because they are needed in a single sheet for manipulation
```

Comment:
why cant we just merge the excel files in python? this is a horrendously expensive query that looks like it can be two much smaller ones (with none of the nasty joining), if we just pull it twice and write a 5 line python script to combine them. If the two queries are ordered correctly, we may even be able to combine without python